src/main/resources/avro/wip/variationReference.avdl

```diff
@@ -0,0 +1,263 @@
+@namespace("org.ga4gh.wip")
```

Comment:
This whole file looks like it's getting added back incorrectly. I removed it when the graph reference stuff was accepted.